exchange/targeting_test.go

```diff
@@ -281,15 +281,23 @@ var bid084 *openrtb2.Bid = &openrtb2.Bid{
 	Price: 0.84,
 }
 
-var truncateTargetAttrValue10 int = 10
-var truncateTargetAttrValue5 int = 5
-var truncateTargetAttrValue25 int = 25
-var truncateTargetAttrValueNegative int = -1
+var (
+	truncateTargetAttrValue10       int = 10
+	truncateTargetAttrValue5        int = 5
+	truncateTargetAttrValue25       int = 25
+	truncateTargetAttrValueNegative int = -1
+)
+
+func lookupPriceGranularity(v string) openrtb_ext.PriceGranularity {
+	priceGranularity, _ := openrtb_ext.NewPriceGranularityFromLegacyID(v)
+	return priceGranularity
+}
```

Comment:
Convenience test method to throw away the second return and use directly for assignments in these test methods.